test/k8sT/Services.go

```diff
@@ -412,6 +412,67 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(func() bool { return !helpers.RunsOnNetNextOr419Kernel() }, "Checks local redirect policy", func() {
```

Comment:
The [GSG](https://docs.cilium.io/en/latest/gettingstarted/local-redirect-policy/) doesn't mention anything about kernel version, neither does the [Kernel version requirements](https://docs.cilium.io/en/latest/operations/system_requirements/#required-kernel-versions-for-advanced-features). Is there a reason we are restricting the versions we run this test on?

Reply:
Quoting @brb "LRP depends on the host-reachable services feature which is enabled only on net-next and 4.19 CI runs."

Reply:
Ok, then might be worth following up to make this clearer in the docs.

Reply:
That's right. I've already updated the documents in the follow-up PR. For the socket lookup call, we need at least kernel version 5.1, but I checked the other day, and the default in GCP seems to be 4.19. So 4.19 as the minimum version for the LRP feature should be okay for now, since we don't have the backend packets getting looped back problem for the node-local dns use case. The reason being, DNS proxy pods while act as backends for `kube-dns` service, they connect to `kube-dns-upstream` service with coreDNS backend pods.

Reply:
It's worth finding out if we need the host-reachable functionality for the node-local dns case. @Weil0ng Can you revert the kernel check, and trigger CI test suite with `test-me-please`/`retest-4.9`?

Reply:
@brb confirmed that the host-reachable functionality is needed for being able to do service translation from host ns. This would apply for pods deployed in the `hostNetwork` mode. So the minimum kernel version required would indeed be 4.19.

Reply:
> So the minimum kernel version required would indeed be 4.19.

To be more precise - 4.19.57 for UDP, and 4.17.0 for TCP.

Reply:
Yes, I've kept the version requirement same as kubeproxy-free/host-services features - https://github.com/cilium/cilium/pull/13362/commits/9289cb65f2b0668addb7a9a19758c42cd4419e93

Reply:
> It's worth finding out if we need the host-reachable functionality for the node-local dns case.

As @aditighag and @brb pointed out, I think for node-local dns, we need host-reachable services too to capture dns requests from host-ns pods.